components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,1198 @@
+
+$$
+\tilde{z}(z) = -\int_z^{z_s} \rho(z') \, dz',
+$$ (def-pseudo-height)
+
+where $z$ is the geometric height (positive upward), $\rho$ is the in-situ density, and $z_s$ is the ocean surface elevation. Pseudo-height $\tilde{z}$ is therefore **zero at the surface** and becomes **increasingly negative downward**, with units of $\mathrm{kg\,m^{-2}}$. The total column mass per unit area is defined as:
+
+$$
+M = \int_{z_b}^{z_s} \rho(z') \, dz' > 0,
+$$ (def-column-mass)
+
+where $z_b$ is the bottom elevation. The bottom of the domain corresponds to:
+
+$$
+\tilde{z}(z_b) = -M.
+$$ (def-total-pseudo-height)
+
+**Note:** Because $z$ appears as the *lower bound* of the integral in equation [](#def-pseudo-height), we have
+$$
+\frac{d\tilde{z}}{dz} = \rho(z), \quad \text{so} \quad d\tilde{z} = \rho(z)\,dz.
+$$
+This means that pseudo-height $\tilde{z}$ increases in the same direction as $z$: both are **zero at the surface** (or close to it) and become **increasingly negative with depth**.
+
+Under hydrostatic balance, we have:
+
+$$
+\frac{dp}{dz} = -\rho g \quad \Rightarrow \quad dp = -\rho g\,dz \quad \Rightarrow \quad \rho\,dz = -\frac{dp}{g},
+$$ (hydrostatic-balance)
+
+which allows us to express pseudo-height in terms of pressure:
+
+$$
+\tilde{z} = -\frac{p - p_s}{g}, \quad \text{or equivalently,} \quad p = p_s - g\tilde{z},
+$$ (pseudo-height-pressure)
+
+where $p_s$ is the pressure at the ocean surface. This makes $\tilde{z}$ a natural coordinate for tracer and mass budgets when $\rho$ is known as a function of pressure, as is the case in TEOS-10, our proposed equation of state.
+
+Given $\rho(\tilde{z})$, we can recover geometric height via:
+
+$$
+z(\tilde{z}) = z_s + \int_{0}^{\tilde{z}} \frac{1}{\rho(\tilde{z}')} \, d\tilde{z}',
+$$ (invert-z-from-pseudo)
+
+noting that the integral moves downward in the column (i.e., toward more negative $\tilde{z}$).
+
+We now formulate the mass and tracer conservation equations in a finite volume using $\tilde{z}$ as the vertical coordinate.
+
+
+
+## 4. Tracer and Mass Equations
+
+
+Here, we closely follow Appendix A2 from Ringler et al. (2013). They state, "The variable $\phi(x, y, z, t)$ may be the fluid density q or the density-weighted concentration of some tracer, in units of tracer mass per volume." This means that $\phi = \rho \varphi$, where $\varphi$ is what we are using as the definition of a tracer.
+
+We begin with their equations (A.10)-(A.13) which are agnostic to our choice of mass-thickness:
+
+$$
+h(x, y, t) = \int_{z^{\text{bot}}}^{z^{\text{top}}} \rho \, dz =  \tilde{z}^{\text{top}}(x, y, t) - \tilde{z}^{\text{bot}} (x, y, t).
+$$ (def-pressure-thickness)
+
+Instead of geometric thickness to define our layers. Starting at their (A.14), it is useful to write the equation in terms of vertical mass flux per unit area $\omega = \rho w$ instead of vertical velocity $w$:
```

Comment:
Would it be consistent to also use \tilde{w} here?

Reply:
Another great suggestion!